ledger/ledger-api-auth/src/main/scala/com/digitalasset/ledger/api/auth/Authorizer.scala

```diff
@@ -305,15 +305,17 @@ final class Authorizer(
   private[auth] def authorizeWithReq[Req, Res](call: Req => Future[Res])(
       authorized: (ClaimSet.Claims, Req) => Either[StatusRuntimeException, Req]
   ): Req => Future[Res] = request =>
-    authenticatedClaimsFromContext() match {
-      case Failure(ex) => Future.failed(ex)
-      case Success(claims) =>
-        authorized(claims, request) match {
-          case Right(modifiedReq) => call(modifiedReq)
-          case Left(ex) =>
-            Future.failed(ex)
-        }
-    }
+    Future.delegate {
```

Comment:
I guess this needs to stay, otherwise this error is produced:
```Thread local context unexpectedly does not store authorization claims. Perhaps a Future was used in some intermediate computation and changed the executing thread?```